api/layer/tree_service.go

```diff
@@ -12,58 +12,58 @@ import (
+	GetBucketTagging(ctx context.Context, cnrID cid.ID) (map[string]string, error)
+	PutBucketTagging(ctx context.Context, cnrID cid.ID, tagSet map[string]string) error
+	DeleteBucketTagging(ctx context.Context, cnrID cid.ID) error
 
-	GetVersions(ctx context.Context, cnrID *cid.ID, objectName string) ([]*data.NodeVersion, error)
-	GetLatestVersion(ctx context.Context, cnrID *cid.ID, objectName string) (*data.NodeVersion, error)
-	GetLatestVersionsByPrefix(ctx context.Context, cnrID *cid.ID, prefix string) ([]*data.NodeVersion, error)
-	GetAllVersionsByPrefix(ctx context.Context, cnrID *cid.ID, prefix string) ([]*data.NodeVersion, error)
-	GetUnversioned(ctx context.Context, cnrID *cid.ID, objectName string) (*data.NodeVersion, error)
-	AddVersion(ctx context.Context, cnrID *cid.ID, newVersion *data.NodeVersion) error
-	RemoveVersion(ctx context.Context, cnrID *cid.ID, nodeID uint64) error
+	GetVersions(ctx context.Context, cnrID cid.ID, objectName string) ([]*data.NodeVersion, error)
+	GetLatestVersion(ctx context.Context, cnrID cid.ID, objectName string) (*data.NodeVersion, error)
+	GetLatestVersionsByPrefix(ctx context.Context, cnrID cid.ID, prefix string) ([]*data.NodeVersion, error)
+	GetAllVersionsByPrefix(ctx context.Context, cnrID cid.ID, prefix string) ([]*data.NodeVersion, error)
+	GetUnversioned(ctx context.Context, cnrID cid.ID, objectName string) (*data.NodeVersion, error)
+	AddVersion(ctx context.Context, cnrID cid.ID, newVersion *data.NodeVersion) error
+	RemoveVersion(ctx context.Context, cnrID cid.ID, nodeID uint64) error
 
-	PutLock(ctx context.Context, cnrID *cid.ID, nodeID uint64, lock *data.LockInfo) error
-	GetLock(ctx context.Context, cnrID *cid.ID, nodeID uint64) (*data.LockInfo, error)
+	PutLock(ctx context.Context, cnrID cid.ID, nodeID uint64, lock *data.LockInfo) error
+	GetLock(ctx context.Context, cnrID cid.ID, nodeID uint64) (*data.LockInfo, error)
 
-	CreateMultipartUpload(ctx context.Context, cnrID *cid.ID, info *data.MultipartInfo) error
-	DeleteMultipartUpload(ctx context.Context, cnrID *cid.ID, multipartNodeID uint64) error
-	GetMultipartUploadsByPrefix(ctx context.Context, cnrID *cid.ID, prefix string) ([]*data.MultipartInfo, error)
-	GetMultipartUpload(ctx context.Context, cnrID *cid.ID, objectName, uploadID string) (*data.MultipartInfo, error)
+	CreateMultipartUpload(ctx context.Context, cnrID cid.ID, info *data.MultipartInfo) error
+	DeleteMultipartUpload(ctx context.Context, cnrID cid.ID, multipartNodeID uint64) error
+	GetMultipartUploadsByPrefix(ctx context.Context, cnrID cid.ID, prefix string) ([]*data.MultipartInfo, error)
+	GetMultipartUpload(ctx context.Context, cnrID cid.ID, objectName, uploadID string) (*data.MultipartInfo, error)
 
 	// AddPart puts a node to a system tree as a child of appropriate multipart upload
 	// and returns objectID of a previous part which must be deleted in NeoFS.
 	// If a part is being added for the first time, the previous part ID will be nil.
```

Comment:
`the previous part ID will be nil` need to be changed, too. It's similar to already mentioned place of interface usage.